fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexOperationConfig.java

```diff
@@ -188,8 +185,16 @@ public static Builder newBuilder() {
         return new Builder();
     }
 
+
+    /**
+     * <em>Deprecated</em>. Synchronized sessions are now automatically determined by the indexing method.
+     * Mutual indexing and index scrubbing (if applicable) do not expect to run exclusively, other indexing methods will
+     * throw an exception if they another active indexing session is detected.
+     * @return always true;
+     */
+    @API(API.Status.DEPRECATED)
     public boolean shouldUseSynchronizedSession() {
-        return useSynchronizedSession;
+        return true;
     }
 
```

Comment:
TeamScale is noting that `shouldUseSynchronizedSession` is not tested, probably worth checking that it always `true` in `testDeprecatedSetUseSynchronizedSession`